src/deckgl-layers/layer-utils/cluster-utils.js

```diff
@@ -41,11 +41,9 @@ export function getGeoJSON(data, getPosition) {
 const clusterResolver = ({clusterRadius}) => `${clusterRadius}`;
 
 const getClusterer = memoize(({clusterRadius, geoJSON}) => {
-  return supercluster({
+  return new Supercluster({
     maxZoom: 20,
     radius: clusterRadius,
-    initial: () => ({points: []}),
-    map: props => props.data,
```

Comment:
why is this deleted without a replacement?

Reply:
breaking change to the library: https://github.com/mapbox/supercluster/releases/tag/v6.0.0